cmd/metadata_prefetch/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# Copyright 2018 The Kubernetes Authors.
```

Comment:
I think we can remove k8s header from new files.

Reply:
We have a check that requires this header to be present, created https://github.com/GoogleCloudPlatform/gcs-fuse-csi-driver/pull/367 to remove this requirement.

Reply:
Will remove when #367 is merged.